Added breaking change note for dropping mysql 5 (#270)
directus/docs: content/releases/3.breaking-changes/2.version-11.md

description: Breaking changes may require action on your part before upgrading.
---

## Version 11.7.0

### Officially dropped support for MySQL 5.7

> [MySQL 5.7 is end of life](https://endoflife.date/mysql) and has not been getting support since 31 Oct 2023.

Reviewer: ```suggestion
[MySQL 5.7 reached end of life](https://endoflife.date/mysql) on 31 October 2023, and is no longer supported.
```
